src/utils/stages.ts

```diff
@@ -1,44 +1,69 @@
-import { ChapterStage, NonprofitStage } from "src/utils/types";
+import { ProjectStage, DisplayableProjectStage } from "src/utils/types";
 
 export const chapterStageOrder = {
```

Comment:
This is a nitpick but the naming is a little off in this file.  Some variable/function names still refer to nonprofit/chapter stages despite the enum name changes so it's a little confusing without knowing the context.